decidim-core/lib/tasks/generators.rake

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "webpush"
+
+namespace :decidim do
```

Comment:
Let's add the `pwa` namespace too, so it makes sense, as adding just a single task is weird, and I can imagine that we will have things like packaging tasks (APKs, IPAs or similar) in the near future. It should be `decidim:pwa:generate_vapid_keys`